### PR TITLE
BHeader (CPDT 90)

### PR DESCRIPTION
**Pull request by ghost-**
Note: We never came to consensus on the name for this component.  I would appreciate any alternatives.

This is the light gray bar before the used for a section/page title.

It accepts its input via slots.

### CODE REVIEW

**Comment by eleanormal** [comment]
So this is more the general page layout, right? Is the slot input supposed to be the content of the page or the title for the page? Or both? If we expect to use this on every page, it seems like that title should be a prop and that it should be an h1. And then this could just be the "Page" component

**Comment by ghost-** [comment]
Yes this is for general page layout.  The thought is the general nav would be part of a boilerplate/template, and this would be used for sections/pages/routes.  

The slot can accept anything.  It can be text or any component/element.  We could convert it into an H1 and make it just accept text if that's whats wanted.

I originally based this on material terminology, even though we are not using material design principles.  The mock-up most closely matched the subheader/secondary toolbar in the material design docs.

I am fine with renaming it "Page", placing it in an H1 and restricting it to text only.  Thoughts?

**Comment by eleanormal** [comment]
Oh, so it's just the toolbar at the top, not meant to contain the whole page. I assumed, with the sides divs, that it would be the whole page. What purpose do the sides serve in the subheader?

**Comment by ghost-** [comment]
Responsiveness for the padding. We could give it a more explicit size though.

File: src/components/BSection/BSection.vue [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<template>
+  <section>
+    <span class="sides"></span>
```
Comment (stevenmwade) [inline src/components/BSection/BSection.vue]: Shouldn't this be solved by having some sort of grid? Adding the sides to a component would seem to take away a lot of flexibility.

Comment (ghost-) [reply src/components/BSection/BSection.vue]: We could implement CSS Grid.  It has less support than flex-box.

https://caniuse.com/#search=grid

Right now this component just needs text and some left/right padding.  We don't really have a use case for anything beyond that yet.

**Comment by eleanormal** [comment]
Ok, this was a very confusing component. It's just a page header with flexible side padding. I don't think that was clear and I'm not sure that it's necessary. If it's just a header, it should use the `<header>` tags, not section. And instead of side spans, if we're looking to add padding, have a `padding` prop.

**Comment by ghost-** [comment]
Sure we can change it.  We can have a fixed padding.

**Comment by ghost-** [comment]
@eleanormal What about the naming? BHeader?

**Comment by eleanormal** [comment]
BHeader works!

**Comment by ghost-** [comment]
Renamed

**Pull request merged**